test/import.bats

```diff
@@ -345,6 +345,23 @@ fourth:
         uid: 1000
         gid: 1000
         dest: /usr/bin
+fifth:
+  from:
+    type: scratch
+  import:
+    - path: /etc/subuid
+      dest: /files/
+    - path: /etc/subgid
+      dest: /files
```

Comment:
you should probably use paths that are created in the test case. I picked /etc/subuid and /etc/subgid because they were kind of known to be there, but for the test better to just create some file paths and use those like is done elsewhere.

Reply:
fixed